### PR TITLE
Precheck network mode/name

### PR DESCRIPTION
**Pull request by sharkconi**
Signed-off-by: Mark.Ma <mark.ma@emc.com>

### CODE REVIEW

**Comment by sharkconi** [comment]
@fub2 @xiar @JuneZhou19

File: infrasim/main.py [hunk 1]
```diff
@@ -82,6 +82,12 @@ def infrasim_main(arg):
     except CommandRunFailed as e:
         print "{} run failed\n".format(e.value)
         print "Infrasim-main starts failed"
+        node.init()
```
Comment (fub2) [inline infrasim/main.py]: :+1:

**Comment by fub2** [comment]
+1

**Comment by XiaowenJiang** [comment]
+1

**Pull request merged**